packages/@webex/plugin-meetings/test/unit/spec/meeting/request.js

```diff
@@ -53,6 +53,11 @@ describe('plugin-meetings', () => {
 
     meetingsRequest.request = request;
     locusDeltaRequestSpy = sinon.spy(meetingsRequest, 'locusDeltaRequest');
+    sinon.stub(Utils, 'anonymizeIPAddress').returns('0.0.0.0');
```

Comment:
issue: This isn't actually doing anything - if it were I would expect tests to be failing due to this always returning `'0.0.0.0'`.

Why is this being `stub`-ed anyway, why not just spy on it, and check that the spy is being used?